docker/entrypoint.sh

```
@@ -33,15 +33,22 @@ else
-php artisan migrate --force
-
 echo -e "Optimizing Filament"
 php artisan filament:optimize
 
 # default to caddy not starting
 export SUPERVISORD_CADDY=false
+export PARSED_LE_EMAIL=${LE_EMAIL}
+export PARSED_APP_URL=${APP_URL}
+
+# when running behind a proxy
+if [[ ${BEHIND_PROXY} == "true" ]]; then
```

Shellcheck (0.11.0)

[error] In dash, [[ ]] is not supported.

(SC3010)